pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -690,6 +722,8 @@ class Build {
                             if (buildConfig.CODEBUILD) {
                                 label = "codebuild"
                             }
+
+                            waitForANodeToBecomeActive(label)
```

Comment:
Can this hang the pipeline forever if no nodes come online?

Reply:
If it all works, no.

It should only be capable of causing us to pause until ACTIVE_NODE_TIMEOUT number of minutes has passed (default 0), or until a node that matches the labels becomes active.